akka-docs/src/main/paradox/typed/index.md

```diff
@@ -0,0 +1,20 @@
+# Akka Typed
```

Comment:
This page was moved, perhaps add a redirect to the `.htaccess` (`src/main/paradox/.htaccess iirc) to make sure links keep working?